<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetNfeSaida</class>
 <widget class="QWidget" name="WidgetNfeSaida">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QLabel" name="labelNfe">
     <property name="text">
      <string>Notas Fiscais</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="TableView" name="table">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonRelatorio">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Gerar Relatório</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButtonExportar">
       <property name="text">
        <string>Exportar XML/PDF</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButtonCancelarNFe">
       <property name="text">
        <string>Cancelar NFe</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonConsultarNFe">
       <property name="text">
        <string>Consultar NFe</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelBusca">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Buscar:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditBusca">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>NFe/Venda/CPF/CNPJ/Cliente</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Filtros</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBoxMes">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Mês</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QDateEdit" name="dateEdit">
           <property name="currentSection">
            <enum>QDateTimeEdit::MonthSection</enum>
           </property>
           <property name="displayFormat">
            <string>MM/yy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBoxStatus">
        <property name="title">
         <string>Status</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_40">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBoxAutorizado">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Autorizado</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBoxCancelado">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Cancelado</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBoxPendente">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Nota Pendente</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>src/tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
